rfc/graphql-schema/additionalTypeDefs/byPage/quotes.ts

```diff
@@ -0,0 +1,99 @@
+export default /* GraphQL */ `
+  type SalesRep implements Node {
+    id: ID!
+    email: String!
+  }
+
+  enum QuoteStatus {
+    DRAFT
+    OPEN
+    ORDERED
+    EXPIRED
+  }
+
+  # at this point the differences between draft/normal are:
+  # - no expiration for draft
+  # - salesRep MIGHT be N/A for draft
+  # which seem small enough to go for optional
+  # over separate types and a union
+  type Quote implements Node {
+    id: ID!
+    status: QuoteStatus!
+    createdBy: Customer!
+    title: String
+    subtotal: Money!
+    createdAt: DateTime!
+    updatedAt: DateTime!
+    expirationDate: DateTime
+    salesRep: SalesRep
+  }
+
+  input QuotesDateRangeFilterInput {
+    from: DateTime
+    to: DateTime
+  }
+
+  input QuotesFiltersInput {
+    searchTerm: String
+    createdBy: ID
+    salesRep: ID
+    status: [QuoteStatus]
+    dateRange: QuotesDateRangeFilterInput
+  }
```

Comment:
separating out `company` vs `customer` quote node filters

```suggestion
  input CompanyQuotesFiltersInput {
    searchTerm: String
    companyUserId: ID
    companyId: [ID!]
    salesRepId: ID
    status: [QuoteStatus]
    dateRange: QuotesDateRangeFilterInput
  }
  
  input QuotesFiltersInput {
    searchTerm: String
    salesRepId: ID
    status: [QuoteStatus]
    dateRange: QuotesDateRangeFilterInput
  }
```